### ISSUE TITLE
Toggle menu 

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/pattern#6]
The content is overlayed on to the styles menu. 

The other issue I now see with this is the horizontal scroll, which makes it feel like it's not responsive.

Why not display the same way Bootstrap does:
- option 1: http://getbootstrap.com/css/
- option 2: http://getbootstrap.com/examples/dashboard/

Overlay bug:

<img width="1102" alt="screen shot 2015-09-22 at 11 39 03 am" src="https://cloud.githubusercontent.com/assets/1569463/10027961/abda0ba8-611e-11e5-9646-10588e8abcfb.png">

**Comment by jlyon** [comment]
Only reason not to is that it would get in the way on the dedicated layouts:
- http://patternlibrary.getproudcity.com/?v=&page=dashboard
- http://patternlibrary.getproudcity.com/?v=&page=01-page

**Comment by lukefretwell** [comment]
WW@aschmoeD?

**Comment by jlyon** [comment]
btw, what browser are you using in the screenshot above? Are you still getting this on pattern.getproudcity.com?

**Comment by lukefretwell** [comment]
Chrome. Getting 503 on pattern.getproudcity.com.

**Comment by jlyon** [comment]
Cool, i think it's probably fixed now.  Sorry, waiting for dns to propagate.

**Comment by jlyon** [comment]
ok http://pattern.getproudcity.com/ has propagated

**Comment by aschmoe** [comment]
@lukefretwell @jlyon Is this issue still active?

**Comment by jlyon** [comment]
Yeah, I think we'll want to do something about this at some point (maybe separate out the pattern library from the full layouts), but it's a lower priority for me at least.
